src/Slider.jsx

```diff
@@ -667,4 +653,31 @@ class Rheostat extends React.Component {
 Rheostat.propTypes = propTypes;
 Rheostat.defaultProps = defaultProps;
 
-export default Rheostat;
+export const RheostatStyles = ({ color, unit }) => ({
```

Comment:
No worries!

I think in any production code, you will want to import both `css` and `withStyles` from `react-with-styles` directly.

Then, for development things (e.g. storybook, tests), you will want a file that initializes react-with-styles with the theme and interface. This shouldn't need to export anything, it just needs to import stuff, call the register functions and be done. This file needs to be imported at some top level for the dev related activities. The readme should include instructions on how to set this up for your own app.

Let me know if this doesn't make sense. I'm also happy to pair on this with you if you prefer.